unittests/web/RPCServerHandlerTests.cpp

```diff
@@ -549,7 +549,7 @@ TEST_F(WebRPCServerHandlerTest, HTTPParamsUnparseableNotArray)
     EXPECT_EQ(session->lastStatus, boost::beast::http::status::bad_request);
 }
 
-TEST_F(WebRPCServerHandlerTest, HTTPParamsUnparseableEmptyArray)
+TEST_F(WebRPCServerHandlerTest, HTTPParamsUnparseableArrayWithDigit)
```

Comment:
These should probably be made into a parameterised test and we check `[]`, `null`, `[{}]`, `[""]`, `""`, `null` and `{}` as success cases, `1` and `[1]` as failures.